app/scripts/services/builds.js

```diff
@@ -2,162 +2,93 @@
+      },
+      cloneBuild: function(buildName, context) {
+        var req = {
+          kind: "BuildRequest",
+          apiVersion: "v1",
+          metadata: {
+            name: buildName
+          }
+        };
+        return DataService.create("builds/clone", buildName, req, context);
+      },
+      isPaused: function(buildConfig) {
+        return annotation(buildConfig, "openshift.io/build-config.paused") === 'true';
+      },
+      canBuild: function(buildConfig) {
+        if (!buildConfig) {
+          return false;
         }
-      );
-    };
 
-    // Function which will 'clone' build from given buildName
-    BuildsService.prototype.cloneBuild = function(buildName, context, $scope) {
-      var req = {
-        kind: "BuildRequest",
-        apiVersion: "v1",
-        metadata: {
-          name: buildName
-        }
-      };
-      DataService.create("builds/clone", buildName, req, context).then(
-        function(build) { //success
-            $scope.alerts = $scope.alerts || {};
-            var logLink = $filter('buildLogURL')(build);
-            var alert = {
-              type: "success",
-              message: "Build " + buildName + " is being rebuilt as " + build.metadata.name + "."
-            };
-            if (logLink) {
-              alert.links = [{
-                href: logLink,
-                label: "View Log"
-              }];
-            }
-            $scope.alerts["rebuild"] = alert;
-        },
-        function(result) { //failure
-          $scope.alerts = $scope.alerts || {};
-          $scope.alerts["rebuild"] =
-            {
-              type: "error",
-              message: "An error occurred while rerunning the build.",
-              details: $filter('getErrorDetails')(result)
-            };
+        if (buildConfig.metadata.deletionTimestamp) {
+          return false;
         }
-      );
-    };
-
-    BuildsService.prototype.isPaused = function(buildConfig) {
-      return $filter('annotation')(buildConfig, "openshift.io/build-config.paused") === 'true';
-    };
-
-    BuildsService.prototype.canBuild = function(buildConfig) {
-      if (!buildConfig) {
-        return false;
-      }
-
-      if (buildConfig.metadata.deletionTimestamp) {
-        return false;
-      }
-
-      if (this.isPaused(buildConfig)) {
-        return false;
-      }
 
-      return true;
-    };
-
-    var annotation = $filter('annotation');
-    BuildsService.prototype.usesDeploymentConfigs = function(buildConfig) {
-      var uses = annotation(buildConfig, 'pipeline.alpha.openshift.io/uses');
-      if (!uses) {
-        return [];
-      }
-
-      try {
-        uses = JSON.parse(uses);
-      } catch(e) {
-        Logger.warn('Could not parse "pipeline.alpha.openshift.io/uses" annotation', e);
-        return;
-      }
-
-      var depoymentConfigs = [];
-      _.each(uses, function(resource) {
-        if (!resource.name) {
-          return;
+        if (this.isPaused(buildConfig)) {
```

Comment:
I originally did that and had `if (isPaused(buildConfig)) {` but then backed out and tested it.  It still works, I believe because the `return { /* methods */ }` is still an instance of an object, its just not build w/a constructor. 

I'll give it one more test spin though just to make sure I didn't miss something.  I probably prefer to do the standalone functions anyway!